assets/JWTCSA/1-jwks.md

```diff
@@ -0,0 +1,38 @@
+---
+search:
+  exclude: true
+---
+
+# JWKS
+
+## Generating a JWKS Key
+
+### Python:jwcrypto
+
+<!-- --8<-- [start:jwcrypto] -->
+
+```python
+import uuid
+from jwcrypto import jwk, jwt
+
+unique_kid = uuid.uuid4()
+
+this_nodes_jwk = jwk.JWK.generate(kty="RSA", size=4096, kid=str(unique_kid))
+
+public_key = this_nodes_jwk.export_public()
+
+# Publish to Datastore in Multinode system
+
+this_token = jwt.JWT(header={"alg": "RS256"},
```

Comment:
This is a JWT but at the same time there is not expiration. There should probably at least be some expiration in the "default example"?